socialhome/streams/app/components/Stream.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="container">
+        <div id="new-content-container" class="hidden">
+            <a href="" id="new-content-load-link" onclick="return false;">
+                <b-badge pill variant="success">
+                    <span id="new-content-count">0</span>
+                    <span>new posts available</span>
+                </b-badge>
+            </a>
+        </div>
+        <div class="grid">
+            <div class="stamped">
+                <stamped-element></stamped-element>
+            </div>
+            <div class="grid-sizer"></div>
+            <div class="gutter-sizer"></div>
+            <stream-element
+                class="grid-item"
+                v-for="content in $store.state.contentList"
+                v-bind="content"
+                :key="content.id"
+                :data-content-id="content.id"
+                :is-user-authentificated="$store.state.isUserAuthentificated"
+                :show-author-bar="$store.state.showAuthorBar"
+                :current-browsing-user-id="$store.state.currentBrowsingUserId"
```

Comment:
The `author-bar` component lists `currentBrowsingProfileId: {type: String, required: true},` but here I think you've not changed it yet.